src/test/java/io/openliberty/tools/intellij/it/UIBotTestUtils.java

```diff
@@ -209,17 +216,22 @@ public static DialogFixture getOpenProjectLocationDialog(CommonContainerFixture
     public static void closeProjectFrame(RemoteRobot remoteRobot) {
         ProjectFrameFixture projectFrame = remoteRobot.find(ProjectFrameFixture.class, Duration.ofSeconds(10));
         // minimize windows os intellij ide to default state
-        if (remoteRobot.isWin()) {
-            minimizeWindow(remoteRobot);
+        if (remoteRobot.isMac()) {
+            projectFrame.clickOnMainMenuWithActions(remoteRobot, "File", "Close Project");
         }
-        clickOnMainMenu(remoteRobot);
-        // Click on File on the Menu bar.
-        ComponentFixture fileMenuEntry = projectFrame.getActionMenu("File", "10");
-        fileMenuEntry.moveMouse();
+        else {
```

Comment:
please add the above comment `// minimize windows os intellij ide to default state` line 218 to here

Reply:
Done